test/test-framework-spec.md

```diff
@@ -154,6 +154,18 @@ The following are the list of available assertions available in the test framewo
    <td>Checks if the specified value is not equal to the expected value.
    </td>
   </tr>
+  <tr>
+     <td>@test:assertExactEquals
+     </td>
+     <td>Checks if the specified entity is exactly equal to the expected entity.
```

Comment:
Should we make this more clear saying both the values refer to the same?

Reply:
Improved function descriptions with b46bdc4